README.md

```diff
@@ -10,44 +10,194 @@ IO operations tool
 [![License](https://img.shields.io/npm/l/io-ops.svg)](https://github.com/teamdigitale/io-ops/blob/master/package.json)
 
 <!-- toc -->
+* [io-ops](#io-ops)
+* [Requirements](#requirements)
+* [Usage](#usage)
+* [Commands](#commands)
+<!-- tocstop -->
 
-- [io-ops](#io-ops)
-- [Usage](#usage)
-- [Commands](#commands)
-  <!-- tocstop -->
+# Requirements
+
+### Environment variables
+The following sets of commands require the environment variabiles listed in the table below:
+`api-services:*`, `users:*`, `migrate-service`.
+
+| Variable name                     | Description                                                                       | type   |
+|-----------------------------------|-----------------------------------------------------------------------------------|--------|
+| BASE_URL_ADMIN                    | The URL of the admin functions API                                                | string |
+| OCP_APIM                          | The key used to authenticate to the admin functions API                           | string |
 
 # Usage
 
 <!-- usage -->
-
 ```sh-session
 $ npm install -g io-ops
 $ io-ops COMMAND
 running command...
 $ io-ops (-v|--version|version)
-io-ops/0.2.0 darwin-x64 node-v10.13.0
+io-ops/0.2.0 darwin-x64 node-v10.14.1
 $ io-ops --help [COMMAND]
 USAGE
   $ io-ops COMMAND
 ...
 ```
-
 <!-- usagestop -->
 
 # Commands
 
 <!-- commands -->
+* [`io-ops api-services:create`](#io-ops-api-servicescreate)
+* [`io-ops api-services:get SERVICEID`](#io-ops-api-servicesget-serviceid)
+* [`io-ops api-services:get-all`](#io-ops-api-servicesget-all)
+* [`io-ops api-services:keys SERVICEID`](#io-ops-api-serviceskeys-serviceid)
+* [`io-ops api-services:keys-regenerate SERVICEID`](#io-ops-api-serviceskeys-regenerate-serviceid)
+* [`io-ops api-services:logo SERVICEID`](#io-ops-api-serviceslogo-serviceid)
+* [`io-ops api-services:update`](#io-ops-api-servicesupdate)
+* [`io-ops hello`](#io-ops-hello)
+* [`io-ops help [COMMAND]`](#io-ops-help-command)
+* [`io-ops messages:attributes`](#io-ops-messagesattributes)
+* [`io-ops messages:check-content`](#io-ops-messagescheck-content)
+* [`io-ops messages:list FISCALCODE`](#io-ops-messageslist-fiscalcode)
+* [`io-ops migrate-service METADATA`](#io-ops-migrate-service-metadata)
+* [`io-ops profiles:delete FISCALCODE`](#io-ops-profilesdelete-fiscalcode)
+* [`io-ops profiles:exist`](#io-ops-profilesexist)
+* [`io-ops profiles:list`](#io-ops-profileslist)
+* [`io-ops services:check`](#io-ops-servicescheck)
+* [`io-ops services:details`](#io-ops-servicesdetails)
+* [`io-ops services:list`](#io-ops-serviceslist)
+* [`io-ops users:create`](#io-ops-userscreate)
+* [`io-ops users:get EMAIL`](#io-ops-usersget-email)
+* [`io-ops users:get-all`](#io-ops-usersget-all)
+* [`io-ops users:subscription EMAIL SUBSCRIPTIONID`](#io-ops-userssubscription-email-subscriptionid)
+* [`io-ops users:update-groups EMAIL`](#io-ops-usersupdate-groups-email)
+
+## `io-ops api-services:create`
```

Comment:
can you please remove all usage (services & users) examples form the readme ? a list of commands suffices as they have inline examples and help

Reply:
I don't think it's possible to remove any subsection of a specific command, because the `Commands` section in the README is generated by the [`oclif-dev readme`](https://github.com/oclif/dev-cli#oclif-dev-readme) command, which does not offer any option to customize it. The only way I can see to remove the examples is to remove them from the code, but it also would remove them from the description provided by the `help` option.